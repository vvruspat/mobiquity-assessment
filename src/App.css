.App {
  max-width: 700px;
  min-height: 100vH;
  margin: auto;
  background: var(--background-color);
}

