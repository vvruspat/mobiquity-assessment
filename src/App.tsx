import "./App.css";

function App() {
  return <div className="App">Mobiquity assessment</div>;
}

export default App;
