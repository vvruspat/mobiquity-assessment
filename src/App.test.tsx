import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders learn react link", () => {
  render(<App />);

  const el = screen.getByTestId("app-container");

  expect(el).toMatchSnapshot();
});
