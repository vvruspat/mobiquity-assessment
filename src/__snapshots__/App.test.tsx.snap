// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="App"
  data-testid="app-container"
>
  <div
    class="page"
  >
    <header
      class="header"
      data-testid="header"
    >
      <div
        class="header_content"
      >
        <svg>
          logo.svg
        </svg>
        <button
          class="button icon-button theme_button"
          data-testid="button"
        >
          <svg>
            dark.svg
          </svg>
        </button>
      </div>
      <div
        class="menu"
        data-testid="menu"
        dir="ltr"
      />
    </header>
    <section
      class="content"
    />
  </div>
</div>
`;
